docs/diagrams/EditCommandActivityDiagram.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+start
+skinparam activityBorderThickness 1.5
+skinparam activity {
+    BorderColor #7F5539
+    BackgroundColor #EDE0D4
+    arrowColor #7F5539
+}
```

Comment:
I like that the formatting for the activity diagram is standardised to the theme colour.